src/components/consul/selectors/DocumentSelector.js

```diff
@@ -37,10 +75,9 @@ export const DocumentSelector = ({ control, field, item }) => {
                 <RegularText>{item.title}</RegularText>
 
                 <TouchableOpacity
-                  onPress={() => {
-                    setDocumentsAttributes(deleteArrayItem(documentsAttributes, index));
-                    setInfoAndErrorText(deleteArrayItem(infoAndErrorText, index));
-                  }}
+                  onPress={() =>
+                    deleteDocumentAlert({ onDeleteDocument, documentId: item.id, index })
```

Comment:
would it be possible to pass the entire method with params directly here, as `documentId` and `index` are only used to call the method in the alert. would `deleteDocumentAlert(() => onDeleteDocument(item.id, index))` work, when changing to the following in the alert definition:

```js
const deleteDocumentAlert = (onPress) {

  ...

  text: texts.consul.startNew.deleteAttributesButtonText,
  onPress,
  style: 'destructive'
```

Reply:
[b2165fe](https://github.com/ikuseiGmbH/smart-village-app-app/pull/540/commits/b2165fe121ef9f8a52809c0490e7fae9b0b7046a)